sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -855,24 +855,32 @@ XMX hardware. Note that these can be returned using
 
 ==== Intel AMX Supported Combinations
 This is currently available in devices with the architecture
-`architecture::intel_cpu_spr`. In this architecture's implementation,
-the type of the C matrix must be the same as the type of the D
-matrix. Therefore, that common type is shown in a single column in the
-table below.
+`architecture::intel_cpu_spr`, and `architecture::intel_cpu_gnr`. In
+this architecture's implementation, the type of the C matrix must be
+the same as the type of the D matrix. Therefore, that common type is
+shown in a single column in the table below. 
 
 [frame="none",options="header"]
 |======================
-| A type | B type | C and D type | M | N | K
-| `matrix_type::uint8`  | `matrix_type::uint8` |
-`matrix_type::sint32`  |  +<=+ 16 |  +<=+ 16 |  +<=+ 64
-| `matrix_type::uint8`  | `matrix_type::sint8` |
-`matrix_type::sint32`  |  +<=+ 16 |  +<=+ 16 |  +<=+ 64
-| `matrix_type::sint8`  | `matrix_type::uint8` |
-`matrix_type::sint32`  |  +<=+ 16 |  +<=+ 16 |  +<=+ 64
-| `matrix_type::sint8`  | `matrix_type::sint8` |
-`matrix_type::sint32`  |  +<=+ 16 |  +<=+ 16 |  +<=+ 64
-|  `matrix_type::bf16`       |  `matrix_type::bf16`   |
+| A type | B type | C and D type | M | N | K | device
+.2+| `matrix_type::uint8`  .2+| `matrix_type::uint8` .2+|
+`matrix_type::sint32`  .2+|  +<=+ 16 .2+|  +<=+ 16 .2+|  +<=+ 64
+|`architecture::intel_gpu_spr`|`architecture::intel_gpu_gnr`
```

Comment:
Looks like no need to divide the cell on two cells:
Currently there is 
```
===================================
| architecture::intel_gpu_spr     |
===================================
| architecture::intel_gpu_gnr     |
===================================
```
Suggesting:
```
===================================
| architecture::intel_gpu_spr     |
| architecture::intel_gpu_gnr     |
===================================
```

Reply:
done